risc0/zkvm/Cargo.toml

```diff
@@ -104,7 +105,7 @@ client = [
 cuda = ["prove", "risc0-circuit-rv32im/cuda", "risc0-zkp/cuda"]
 dual = []
 metal = ["prove", "risc0-circuit-rv32im/metal", "risc0-zkp/metal"]
-default = ["client", "prove"]
+default = ["client", "prove", "json"]
```

Comment:
I think we can avoid a new feature flag since the `loop` is an example, which falls under `dev-dependencies`. I think it's fine for `dev-dependencies` to depend on `serde_json` without it being optional.